src/resolve/resolved-varlink.c

```diff
@@ -551,6 +593,39 @@ int manager_varlink_init(Manager *m) {
                 return log_error_errno(r, "Failed to attach varlink connection to event loop: %m");
 
         m->varlink_server = TAKE_PTR(s);
+
+        if (m->varlink_notification_server)
+                return 0;
+
+        r = varlink_server_new(&s, VARLINK_SERVER_ACCOUNT_UID);
```

Comment:
why is the server being created if the notifications are not enabled?

Reply:
I fixed this in the latest patch